addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -479,13 +488,17 @@ snippetOptions.registry.gallery_img = snippetOptions.SnippetOptionWidget.extend(
      *
      * @override
      */
-    onRemove: function () {
-        this.trigger_up('option_update', {
-            optionName: 'gallery',
-            name: 'image_removed',
-            data: {
-                $image: this.$target,
-            },
+    onRemove: async function (context) {
+        await new Promise((resolve) => {
```

Comment:
Guys, it's cool to now allow `onRemove` to be async (as it was our long term plan for all `onXXX` methods) but maybe it should be documented in the original function? :wink: Like onTargetShow, onTargetHide